pkg/config/config.go

```diff
@@ -522,3 +525,47 @@ func GetSrcType(ctStr string) (SrcType, error) {
 		return NONE, fmt.Errorf("Unsupported component source type: %s", ctStr)
 	}
 }
+
+// CorrectSourcePath corrects the current sourcePath depending on local or binary configuration,
+// if Git has been passed, we simply return the source location from LocalConfig
+// this will get the correct source path whether on Windows, macOS or Linux.
+func CorrectSourcePath(localConfig *LocalConfigInfo) (path string, err error) {
```

Comment:
Can we call this function something else? It sounds like we are correcting something that is broken :-(
Also to align it with the others wouldn't it be better to have LocalConfigInfo as a receiver?

Reply:
That makes more sense :+1: